services/app-web/src/components/ProgramSelect/ProgramSelect.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import styles from './ProgramSelect.module.scss'
+import Select, { AriaOnFocus, MultiValue, ActionMeta } from 'react-select'
+import { Program } from '../../gen/gqlClient'
+
+export type ProgramSelectPropType = {
+    statePrograms: Program[]
+    programIDs: string[]
+    onChange: (
+        newValue: MultiValue<ProgramOption>,
+        actionMeta: ActionMeta<ProgramOption>
+    ) => string[] | []
+}
+
+interface ProgramOption {
+    readonly value: string
+    readonly label: string
+    readonly isFixed?: boolean
+    readonly isDisabled?: boolean
+}
+
+export const ProgramSelect = ({
+    statePrograms,
```

Comment:
This component's props should be more extendable to be re-useable across the application. We should be able to pass in more the HTML attributes (aria labels, custom ids, etc) and override the defaults-  see `FieldTextInput` as an example.